src/cassandra/session.rs

```diff
@@ -135,6 +137,19 @@ impl Session {
         unsafe { <CassFuture<CassResult>>::build(cass_session_execute(self.0, statement.inner())) }
     }
 
+    /// Execute a statement and get any custom payloads.
```

Comment:
Can you elaborate slightly in the comment, and say "get any custom payloads _from the response_"?

Reply:
Yes